newIDE/app/src/Utils/ProjectErrorsChecker.js

```diff
@@ -49,58 +49,58 @@ export const getProjectPropertiesErrors = (
     addError(
       'packageName',
       'error',
-      t('The package name is empty.'),
-      t('Choose and enter a package name in the game properties.')
+      i18n._(t`The package name is empty.`),
+      i18n._(t`Choose and enter a package name in the game properties.`)
     );
   } else if (project.getPackageName().length >= 255) {
     addError(
       'packageName',
       'error',
-      t('The package name is too long.'),
-      t('Change the package name in the game properties.')
+      i18n._(t`The package name is too long.`),
+      i18n._(t`Change the package name in the game properties.`)
     );
   } else if (!validatePackageName(project.getPackageName())) {
     addError(
       'packageName',
       'error',
-      t(
-        'The package name is containing invalid characters or not following the convention "xxx.yyy.zzz" (numbers allowed after a letter only).'
+      i18n._(
+        t`The package name is containing invalid characters or not following the convention "xxx.yyy.zzz" (numbers allowed after a letter only).`
       ),
-      t('Change the package name in the game properties.')
+      i18n._(t`Change the package name in the game properties.`)
     );
   }
 
   if (!project.getName()) {
     addError(
       'name',
       'error',
-      t('The name of your game is too long.'),
-      t('Change the name in the game properties.')
+      i18n._(t`The name of your game is too long`),
+      i18n._(t`Change the name in the game properties.`)
     );
   }
 
   if (!project.getVersion() || project.getVersion().split('.').length < 3) {
     addError(
       'packageName',
       'error',
-      t("The version that you've set for the game is invalid."),
-      t('Enter a version in the game properties.')
+      i18n._(t`The version that you've set for the game is invalid.`),
+      i18n._(t`Enter a version in the game properties.`)
     );
   }
 
   return errors;
 };
 
 export const displayProjectErrorsBox = (
-  t: TFunction,
```

Comment:
An idea why the TFunction was passed as a parameter before rather than being imported?

Reply:
I'm not sure. it was even not a real TFunction but a fake one.
I'm assuming this was multiple years old and never translated properly.